qa/L0_large_payload/large_payload_test.py

```diff
@@ -117,15 +94,20 @@ def _test_helper(self,
             with self.assertRaises(InferenceServerException):
                 results = client[1].infer(model_name, inputs)
 
-        if client[0] == grpcclient:
-            inputs = [
-                client[0].InferInput(input_name, self._very_large_in0.shape,
-                                     np_to_triton_dtype(self._data_type))
-            ]
-            inputs[0].set_data_from_numpy(self._very_large_in0)
-            # GRPC must fail for large payloads because of a 2GB protobuf limit
-            with self.assertRaises(InferenceServerException):
-                results = client[1].infer(model_name, inputs)
+        # FIXME Test is terminated due to libprotobuf FATAL error when GRPC sends
+        # the second request with input tensors larger than 1.3GBs. In this test 
+        # GRPC has been currently exempted from testing for Very Large tensor(3GBs)
+        # until problem is resolved. Should be uncommented once GRPC is fixed.
```

Comment:
until **the** problem
Should be uncommented once **the GRPC issue is resolved**.

Reply:
Updated